test/functionalTest/cases/674_servicepath_in_notifications/service_path_in_ontimeinterval_notification.test

```diff
@@ -106,8 +106,8 @@ echo
 
 echo "05. Query accumulator"
 echo "====================="
-sleep 2
-curl localhost:${LISTENER_PORT}/dump -s -S
+# sleep 2
```

Comment:
It's the exact same thing.
Not sure it is safe to remove these sleeps (that was not depending on valgrind.
A few tests are not enough to assure that it really works